### PR TITLE
Tutorials on Redis and Jenkins

### PR DESCRIPTION
**Pull request by ajeetraina**

### CODE REVIEW

File: docs/create/jenkins/index-jenkins.mdx [hunk 1]
```diff
@@ -0,0 +1,335 @@
+---
+id: index-jenkins
+title: How to Deploy a Redis Enterprise Database from a Jenkins Pipeline 
+sidebar_label: Redis using Jenkins
+slug: /create/jenkins
+authors: [ajeet, matthew]
+---
+
+
+[Jenkins](https://www.jenkins.io/) is currently [the most popular CI tool](https://cd.foundation/announcement/2019/08/14/jenkins-celebrates-15-years/), with ~15M users. It is an open source automation server which enables developers to reliably build, test, and deploy their software. It was forked in 2011 from a project called Hudson after a[ dispute with Oracle](https://www.infoq.com/news/2011/01/jenkins/), and is used for[ Continuous Integration and Continuous Delivery (CI/CD)](https://stackoverflow.com/questions/28608015/continuous-integration-vs-continuous-delivery-vs-continuous-deployment) and test automation. Jenkins is based on Java and provides over [1700 plugins](https://plugins.jenkins.io/) to automate your developer workflow and save a lot of your time in executing your repetitive tasks.
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
[Jenkins](https://www.jenkins.io/) is currently [the most popular CI tool](https://cd.foundation/announcement/2019/08/14/jenkins-celebrates-15-years/), with ~15M users. It is an open source automation server which enables developers to reliably build, test, and deploy their software. It was forked in 2011 from a project called Hudson after a [dispute with Oracle](https://www.infoq.com/news/2011/01/jenkins/), and is used for [Continuous Integration and Continuous Delivery (CI/CD)](https://stackoverflow.com/questions/28608015/continuous-integration-vs-continuous-delivery-vs-continuous-deployment) and test automation. Jenkins is based on Java and provides over [1700 plugins](https://plugins.jenkins.io/) to automate your developer workflow and save a lot of your time in executing your repetitive tasks.
```

File: docs/create/jenkins/index-jenkins.mdx [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+
+
+![image](images/image1.png)
+
+
+[Source:](https://www.datanyze.com/market-share/ci--319) Datanyze market analysis
+
+[Jenkins Pipeline](https://www.jenkins.io/solutions/pipeline/) performs Continuous Delivery tasks declared in a `Jenkinsfile` stored alongside code. The [Pipeline plugin](https://plugins.jenkins.io/workflow-aggregator) has a fairly comprehensive[ tutorial](https://github.com/jenkinsci/pipeline-plugin/blob/master/TUTORIAL.md) checked into its source tree. Using Pipeline, you can configure Jenkins to automatically deploy key pieces of infrastructure, such as a Redis database.
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
[Jenkins Pipeline](https://www.jenkins.io/solutions/pipeline/) performs Continuous Delivery tasks declared in a `Jenkinsfile` stored alongside code. The [Pipeline plugin](https://plugins.jenkins.io/workflow-aggregator) has a fairly comprehensive [tutorial](https://github.com/jenkinsci/pipeline-plugin/blob/master/TUTORIAL.md) checked into its source tree. Using Pipeline, you can configure Jenkins to automatically deploy key pieces of infrastructure, such as a Redis database.
```

File: docs/create/jenkins/index-jenkins.mdx [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+- [redis-standard-size-s.json.template](https://github.com/masyukun/redis-jenkins-pipeline/blob/main/redis-standard-size-s.json.template)
+- [redis-standard-size-xl.json.template](https://github.com/masyukun/redis-jenkins-pipeline/blob/main/redis-standard-size-xl.json.template)
+
+
+##  Configuring Jenkins
+
+
+### Installing Jenkins
+
+You can use[ Docker Desktop](https://www.docker.com/products/docker-desktop) to quickly get a Jenkins instance up and running, exposing ports 8080 (web GUI) and 50000 (inbound agents).
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
You can use [Docker Desktop](https://www.docker.com/products/docker-desktop) to quickly get a Jenkins instance up and running, exposing ports 8080 (web GUI) and 50000 (inbound agents).
```

File: docs/create/jenkins/index-jenkins.mdx [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+}
+```
+
+
+Click "Save" when the job spec is complete.
+
+
+### Run the Jenkins pipeline
+
+Click on the pipeline you created
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
Click on the pipeline you created:
```

File: docs/create/jenkins/index-jenkins.mdx [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+Sample output: you should see a verbose response from Redis’s REST service in the “Shell Script” accordion pane.
+
+There’s also a “Git” output log, in case you need to debug something at that level. Any time you update the branch in the remote git repository, you should see evidence in that log that the latest changes have successfully checked out into the local Jenkins git repository.
+
+![alt_text](images/image19.png "image_tooltip")
+
+
+
+
+Open your Redis Enterprise Secure Management UI at 'https://servername:8443 and click on the **databases** menu item to verify that your database was created with the name, port, and size specified in the `deployment-configuration-file.json` file.
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
Open your Redis Enterprise Secure Management UI at `https://servername:8443` and click on the **databases** menu item to verify that your database was created with the name, port, and size specified in the `deployment-configuration-file.json` file.
```

File: docs/create/jenkins/index-jenkins.mdx [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+
+![alt_text](images/image20.png "image_tooltip")
+
+
+
+Congratulations! You have deployed a Redis Enterprise database using a Jenkins Pipeline!
+
+The GitHub repository is currently:[ https://github.com/masyukun/redis-jenkins-pipeline](https://github.com/masyukun/redis-jenkins-pipeline)
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
The GitHub repository is currently: [https://github.com/masyukun/redis-jenkins-pipeline](https://github.com/masyukun/redis-jenkins-pipeline)
```

File: docs/create/jenkins/index-jenkins.mdx [hunk 7]
```diff
@@ -0,0 +1,335 @@
...
+
+
+### Further Read
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
### Further Reading
```

File: docs/create/jenkins/index-jenkins.mdx [hunk 8]
```diff
@@ -0,0 +1,335 @@
...
+
+
+
+- [How to Embed Redis into Your CI-CD Process](https://redis.com/blog/how-to-embed-redis-into-your-continuous-integration-and-continuous-deployment-ci-cd-process/)
+- [Top Redis Headache for DevOps](https://redis.com/blog/top-redis-headaches-for-devops-replication-timeouts/)
```
Comment (simonprickett) [inline docs/create/jenkins/index-jenkins.mdx]:
```suggestion
- [Top Redis Headaches for DevOps](https://redis.com/blog/top-redis-headaches-for-devops-replication-timeouts/)
```

**Review by simonprickett — APPROVED** [review]
@ajeetraina approved - left some minor suggestions for a couple typos and link formatting.

**Comment by ajeetraina** [comment]
Thanks @simonprickett for your quick review.

**Pull request merged**